src/NearOrg/Notifications/utils.jsx

```diff
@@ -105,6 +114,17 @@ function createNotificationLink(notificationType, notificationValue, authorAccou
           id: post,
         },
       });
+    case "devhub/mention":
+    case "devhub/edit":
+    case "devhub/reply":
+    case "devhub/like":
+      return href({
+        widgetSrc: "devhub.near/widget/app",
+        params: {
+          page: "proposal",
+          id: proposal,
+        },
```

Comment:
@Megha-Dev-19 I think you also need to pass the [`notifier` to the parameters](https://github.com/NEAR-DevHub/neardevhub-contract/commit/46ca5c3f2b469e1a56b5773ae6224928b51a1649)

Reply:
Oh I didn't push the commit, just did, update the profile as well